packages/components/pager/src/pager.js

```diff
@@ -0,0 +1,199 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './pager.css';
+import { customEventNames, perPageSelectValues } from './utils';
+import translations from './utils/translations';
+
+import '@tradeshift/elements.icon';
+import '@tradeshift/elements.tooltip';
+
+customElementDefineHelper(
+	'ts-pager',
+	class extends TSElement {
+		constructor() {
+			super();
+			this.activePage = 1;
+			this._translations = Object.assign({}, translations);
+		}
+
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				totalPages: { type: Number, reflect: true, attribute: 'total-pages' },
+				activePage: { type: Number, reflect: true, attribute: 'active-page' },
+				perPage: { type: Number, reflect: true, attribute: 'per-page' },
+				translations: { type: Object, reflect: true }
+			};
+		}
+
+		get translations() {
+			return this._translations;
+		}
+
+		set translations(val) {
+			let oldVal = this._translations;
+			this._translations = Object.assign(oldVal, val);
+			this.requestUpdate('translations', oldVal);
+		}
+
+		get toStart() {
+			return this.navigationButton({
+				className: 'first-page',
+				disabled: this.activePage === 1,
+				tooltip: this.translations.first_page_tooltip,
+				icon: html`
+					<ts-icon icon="arrow-left-skip" size="large"></ts-icon>
+				`,
+				callback: () => (this.activePage = 1)
+			});
+		}
+
+		get toEnd() {
+			return this.navigationButton({
+				className: 'last-page',
+				tooltip: this.translations.last_page_tooltip,
+				icon: html`
+					<ts-icon icon="arrow-left-skip" size="large" flip="h"></ts-icon>
+				`,
+				disabled: this.activePage === this.totalPages,
+				callback: () => (this.activePage = this.totalPages)
+			});
+		}
+
+		get prev() {
+			return this.navigationButton({
+				className: 'prev-page',
+				disabled: this.activePage === 1,
+				tooltip: this.translations.first_page_tooltip,
+				icon: html`
+					<ts-icon icon="arrow-down-short" size="large" rotate="90"></ts-icon>
+				`,
+				callback: () => this.activePage--
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this decrement operation into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS881)